test/service.go

```diff
@@ -16,17 +16,18 @@ import (
 	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
 )
 
-func Service(name, namespace, image string, annotations map[string]string) *servingv1.Service {
+func Service(name, namespace, image string, ksvcLabels map[string]string, revisionTemplateAnnotations map[string]string) *servingv1.Service {
```

Comment:
Umm... I feel that argument is too many. How about adding label in servicemesh_test.go like this?

```
svc := test.Service(scenario.name, serviceMeshTestNamespaceName, serviceMeshTestImage, scenario.annotations))
svc.ObjectMeta.Labels = scenario.labels
```

This is just a preference, though.

Reply:
I tend to agree.